### ISSUE TITLE
Unable to get SimpleX-smp Server address

### ISSUE THREAD

**Issue by jacobthetechy** [meienberger/runtipi-appstore#912]
From the description

To get the SMP Server Address

SSH into your Tipi Server
Run `sudo docker logs simplex-smp 2>&1 | grep "Server address:"` to find out the SMP Server Address.

`sudo docker logs simplex-smp 2>&1 | grep "Server address:"` returns nothing.

**Comment by DrMxrcy** [comment]
Hello! Did you give the app a minute to start up? I just ran it and was able to get the server address.

**Comment by jacobthetechy** [comment]
@DrMxrcy  Thanks for asking but I still was having an issue. 

I ended up doing a fresh install of Runtip, had a separate issue that needed it anyway, and that seemed to work. The original install was from quite a old version of Runtipi so I wondering if that could have been the root cause. 

Tho I did run into a issue of having an 'option --password: Failed reading: invalid character in BasicAuth` when setting the password to `this is a test`. I'm assuming the "space" is what is causing an issue. 

A secondary issue I have found is if you update the password and restart the service that does not change the server address. Not sure if that is an issue or not since I'm very new to this project.

Again thanks for asking!
